source/services/test/chaiMoment.tests.ts

```diff
@@ -0,0 +1,34 @@
+import * as moment from 'moment';
+import 'moment-timezone';
+
+import './test.module';
+
+describe('chai moment', () => {
+	it('should return true if moments are equal', (): void => {
+		expect(moment('1/1/2015')).to.be.sameMoment(moment('1/1/2015'));
```

Comment:
They're aliases. The chai-moment `npm` package uses `sameMoment`, but I thought `equalMoment` sounded better.